hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -679,5 +687,4 @@ private String appendSlashIfNeeded(String authority) {
     }
     return authority;
   }
-
```

Comment:
Avoid.

Reply:
@goiri I will fix this in the next update.